pkg/kubelet/dockershim/docker_image_linux.go

```diff
@@ -20,12 +20,58 @@ package dockershim
 
 import (
 	"context"
-	"fmt"
+	"path/filepath"
+	"os"
+	"time"
+
+	"k8s.io/klog"
 
 	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
 // ImageFsInfo returns information of the filesystem that is used to store images.
 func (ds *dockerService) ImageFsInfo(_ context.Context, r *runtimeapi.ImageFsInfoRequest) (*runtimeapi.ImageFsInfoResponse, error) {
```

Comment:
Nit: Can we do `_ *runtime.ImageFs...`? I don't think we actually use the request parameter anywhere.